schema.json

```diff
@@ -11,6 +11,10 @@
             "id": {
               "type": "integer"
             },
+            "caipId": {
+              "type": "string",
+              "pattern": "^eip155:\\d+$"
```

Comment:
This is bad regex, as it doesn't need to be `eip155`. You can remove regex for now tbh.